source/runner/Executor.ts

```diff
@@ -84,6 +84,7 @@ export class Executor {
         console.log("Found some message, writing it down")
       }
       const comment = githubResultsTemplate(results)
+      // TODO filter out Danger items that contain file/line info
```

Comment:
This function only accepts the `results: DangerResults`. I'm thinking we will need the `context: DangerContext` (or at least `danger: DangerDSLType`), so that we can call `danger.git.diffForFile(violation.options.file)` and determine if the `{ file, line }` supplied are valid for creating an inline comment. Does that make sense?